src/cli/utils/cli-helper.js

```diff
@@ -118,6 +127,11 @@ export const desc = {
   claimObj: (objName) => `claim a ${objName} that is not COMPLETED`,
   deposit: () => 'deposit RLC onto your iExec account (default unit nRLC)',
   withdraw: () => 'withdraw RLC from your iExec account (default unit nRLC)',
+  approve: () =>
+    'approve spender to spend up to amount of RLC from your iExec account (default unit nRLC)',
+  allowance: () =>
+    "checks the amount of allowance approved for the specified spender to use your iExec account (specify --user to see another user's allowance)",
```

Comment:
Minor: To match other descriptions:

```suggestion
    "check the amount of allowance approved for the specified spender to use your iExec account (specify --user to see another user's allowance)",
```